<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserWindow</class>
 <widget class="QMainWindow" name="BrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>512</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/gui.qrc">
    <normaloff>:/gui/logo_v2.png</normaloff>:/gui/logo_v2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionReload"/>
   <addaction name="actionStop"/>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-shutdown-64.png</normaloff>:/gui/icons8-shutdown-64.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Home</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-sync-64.png</normaloff>:/gui/icons8-sync-64.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-back-arrow-64.png</normaloff>:/gui/icons8-back-arrow-64.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-forward-button-64.png</normaloff>:/gui/icons8-forward-button-64.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/icons8-cancel-64.png</normaloff>:/gui/icons8-cancel-64.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/gui.qrc"/>
 </resources>
 <connections/>
</ui>
